README.md

```diff
@@ -1,31 +1,32 @@
-# DeFi Wonderland's Solhint Plugin
+# Lido Community Staking Module Solhint Plugin
 
-[![image](https://img.shields.io/npm/v/solhint-plugin-defi-wonderland.svg?style=flat-square)](https://www.npmjs.org/package/solhint-plugin-defi-wonderland)
-![Tests](https://github.com/defi-wonderland/solhint-plugin/actions/workflows/unit-tests.yml/badge.svg)
-[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/defi-wonderland/solhint-plugin/blob/main/LICENSE)
+[![image](https://img.shields.io/npm/v/solhint-plugin-lido-csm.svg?style=flat-square)](https://www.npmjs.org/package/solhint-plugin-lido-csm)
+![Tests](https://github.com/lidofinance/solhint-plugin-lido-csm/actions/workflows/unit-tests.yml/badge.svg)
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/lidofinance/solhint-plugin-lido-csm/blob/main/LICENSE)
 
 ## Overview
 
-This custom Solhint Plugin aims to contain all of the Solidity best practices implemented at [DeFi Wonderland](https://defi.sucks).
+This custom Solhint Plugin aims to contain all of the Solidity best practices and style guide of Community Staking
```

Comment:
Add a note about the initial version by Wonderland as credits

Reply:
Done